c2corg_ui/templates/outing/edit.html

```diff
@@ -367,14 +367,18 @@ <h2 class="heading show-phone"><span translate>conditions</span></h2>
             </div>
 
             ## AVALANCHE SIGNS
-            <div class="form-group half" ng-class="{'has-success': outing.avalanche_signs}"
+            <div class="form-group half" ng-init="signs = ${avalanche_signs}; outing.avalanche_signs = outing.avalanche_signs || [];"
                   ng-if="outing.activities.indexOf('skitouring') > -1 || outing.activities.indexOf('snowshoeing') > -1 || outing.activities.indexOf('ice_climbing') > -1 || outing.activities.indexOf('snow_ice_mixed') > -1">
               <label translate>avalanche_signs</label>
-              <div class="input-group">
-                <select class="form-control" ng-model="outing.avalanche_signs" ng-options="rat as rat for rat in ${avalanche_signs}"><option value=""></option></select>
-                <span class="input-group-addon"><span class="glyphicon glyphicon glyphicon-alert"></span></span>
-              </div>
-              <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="outing.avalanche_signs"></span>
+              <ul class="types">
+                <li ng-click="outing.avalanche_signs.indexOf('no') == -1 ? (outing.avalanche_signs = ['no']) : (outing.avalanche_signs[outing.avalanche_signs.indexOf('no')] = null)">
+                  <div class="checkbox"><input type="checkbox" ng-checked="outing.avalanche_signs.indexOf('no') > -1"><span>{{'no' | translate}}</span></div>
+                </li>
+                ## don't show other options if 'no' is selected
```

Comment:
What if the user has checked other options before checking "no"? The other options will be submitted as well, won't they?